<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>1</y>
     <width>598</width>
     <height>614</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="ReloadButton">
          <property name="text">
           <string>Отменить фильтры</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="FilterButton">
          <property name="text">
           <string>Просмотр</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="FiltersCombo"/>
        </item>
        <item>
         <widget class="QGroupBox" name="gbBorderPolicy">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Настройка краев</string>
          </property>
          <layout class="QVBoxLayout" name="_3">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>9</number>
           </property>
           <item>
            <widget class="QRadioButton" name="rbExtend">
             <property name="text">
              <string>Растягивать</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbMirror">
             <property name="text">
              <string>Отражать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbWrap">
             <property name="text">
              <string>Заворачивать</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbChannels">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Каналы</string>
          </property>
          <layout class="QVBoxLayout" name="_4">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>9</number>
           </property>
           <item>
            <widget class="QCheckBox" name="ckRed">
             <property name="text">
              <string>Красный</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ckGreen">
             <property name="text">
              <string>Зеленый</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ckBlue">
             <property name="text">
              <string>Синий</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ckAlpha">
             <property name="text">
              <string>Альфа</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbMirror">
          <property name="title">
           <string>Параметры зеркала</string>
          </property>
          <layout class="QVBoxLayout" name="_5">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>9</number>
           </property>
           <item>
            <widget class="QCheckBox" name="ckHorizontal">
             <property name="text">
              <string>Горизонтально</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ckVertical">
             <property name="text">
              <string>Вертикально</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="PixmapLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string>Превью</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>596</width>
        <height>32</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Применить к холсту</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
